samples/projection-management/Program.cs

```diff
@@ -0,0 +1,312 @@
+			await managementClient.ResetAsync("countEvents_Reset");	
+			// It is safe to call abort multiple times 
+			await managementClient.ResetAsync("countEvents_Reset");
+			#endregion Reset
+
+		}
+
+		private static async Task CreateOneTime(EventStoreProjectionManagementClient managementClient) {
+
+			const string js =
+				"fromAll() .when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+			await managementClient.CreateOneTimeAsync(js);
+
+		}
+
+		private static async Task CreateContinuous(EventStoreProjectionManagementClient managementClient) {
+
+			#region CreateContinuous
+			try {
+				var js =
+					"fromAll() .when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+				await managementClient.CreateContinuousAsync("countEvents_Create", js);
+
+
+			} catch (InvalidOperationException e) when (e.Message.Contains("Conflict")) {
+				Console.WriteLine($"countEvents_Create already exists");
+			}
+			#endregion CreateContinuous
+
+
+		}
+
+
+
+		private static async Task Update(EventStoreProjectionManagementClient managementClient) {
+
+			#region Update
+			const string js = "fromAll().when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+			var name = $"countEvents_Update_{Guid.NewGuid()}";
+
+			await managementClient.CreateContinuousAsync(name, "fromAll().when()");
+			await managementClient.UpdateAsync(name, js);
+			
+			
+			try {
+				await managementClient.UpdateAsync("countEvents_Update_Not_existing", js);
+			} catch (InvalidOperationException e) when (e.Message.Contains("NotFound")) {
+				
+				Console.WriteLine("countEvents_Update_Not_existing does not exists and can not be updated");
+			}
+			#endregion Update
+		}
+
+		private static async Task ListAll(EventStoreProjectionManagementClient managementClient) {
+			#region ListAll
+			var details = managementClient.ListAllAsync();
+			await foreach (var detail in details) {
+				Console.WriteLine(
+					$@"{detail.Name}, {detail.Status}, {detail.CheckpointStatus}, {detail.Mode}, {detail.Progress}");
+			}
+			#endregion ListAll
+		}
+		
+		
+		private static async Task ListContinuous(EventStoreProjectionManagementClient managementClient) {
+			#region ListContinuous
+			var details = managementClient.ListContinuousAsync();
+			await foreach (var detail in details) {
+				Console.WriteLine(
+					$@"{detail.Name}, {detail.Status}, {detail.CheckpointStatus}, {detail.Mode}, {detail.Progress}");
+			}
+			#endregion ListContinuous
+		}
+
+		private static async Task GetStatus(EventStoreProjectionManagementClient managementClient) {
+			
+			#region GetStatus
+			const string js = "fromAll().when({$init:function(){return {count:0};},$any:function(s, e){s.count += 1;}}).outputState();";
+			var name = $"countEvents_status_{Guid.NewGuid()}";
+
+			await managementClient.CreateContinuousAsync(name,js);
```

Comment:
**SUGGESTION:** I think that we may just keep the `GetStatusAsync` usage. Repeating creation in each sample seems to be a bit redundant.

Reply:
it's to keep them selfcontained .
this is meant for the doc snippets .

Reply:
I understand, but other snippets that we have do not repeat all the setup unless it's crucial for understanding the use case.

Reply:
also get state snippet is not imported right now due to a bug in the client #142